### DISCUSSION TITLE
An error was reported when UE4 was loaded to 75%

### DISCUSSION THREAD

**Discussion by Meyond-LZY** [carla-simulator/carla#7637] [Q&A]
![微信图片_20240513194354](https://github.com/carla-simulator/carla/assets/95667439/6391b680-decc-468d-9887-808047c94cfd)
模块"CarlaTools"无法被加载，因此插件"CarlaTools"加载失败，可能存在系统错误，或模块未能正确设置，

**Comment by Aiden885** [comment]
我也有类似的错误，请问现在解决了吗
